### PR TITLE
fixes error handling

### PR DESCRIPTION
**Pull request by usersadvocate**

### CODE REVIEW

File: frontend/src/app/shared/components/account/account.component.ts [hunk 1]
```diff
@@ -99,7 +99,7 @@ export class AccountComponent implements OnInit {
               this.accountModalService.openWelcomeDialog();
             }
         });
-    this.signupForm.reset();
+    //this.signupForm.reset();
```
Comment (MatvienkoDO) [inline frontend/src/app/shared/components/account/account.component.ts]: there is sense to just delete those strings of code

File: frontend/src/app/shared/components/account/account.component.ts [hunk 2]
```diff
@@ -126,13 +126,8 @@ export class AccountComponent implements OnInit {
               this.accountModalService.openWelcomeDialog();
             }
         });
-    this.loginForm.reset();
+    //this.loginForm.reset();
```
Comment (MatvienkoDO) [inline frontend/src/app/shared/components/account/account.component.ts]: same

**Review by linkouth — APPROVED** [review]

File: frontend/src/app/core/interceptors/error-handler.interseptor.ts [hunk 3]
```diff
@@ -61,8 +61,10 @@ export class ErrorInterceptor implements HttpInterceptor {
             this.errorsService.pushError(Error.Target.SignUp, info.message);
 
         } else if (info.status === constants.incorrectLoginPasswordPair) {
-          const message = this.getNotificationServiceMessage('invalidLoginPasswordPair');
-          this.errorsService.pushError(Error.Target.LogIn, message);
+          this.notificationService.error(
+            this.getNotificationServiceMessage('incorrectLoginPasswordPair'));
+            this.errorsService.pushError(Error.Target.SignUp, info.message);
```
Comment (MatvienkoDO) [inline frontend/src/app/core/interceptors/error-handler.interseptor.ts]: Don't understand why it is needed. You've changed channel of notifications

Comment (MatvienkoDO) [inline frontend/src/app/core/interceptors/error-handler.interseptor.ts]: too much weird things

**Review by MatvienkoDO — APPROVED** [review]

**Pull request merged**